README.md

```diff
@@ -498,6 +504,12 @@ const tenantsRes = await descopeClient.management.tenant.loadAll();
 tenantsRes.data.forEach((tenant) => {
   // do something
 });
+
+// Search all tenants according to various parameters
+const tenantsRes = await descopeClient.management.tenant.searchAll(['id']);
```

Comment:
```suggestion
const tenantsRes = await descopeClient.management.tenant.searchAll(['tenant-id']);
```

not critical ofc, just communicating the data purpose

Reply:
I disagree here
Since we are in tenant scope, it is clear that this is tenant ID 
Unlike when we are in users, and look for tenant ID, there we need to mention that this is a tenant id